src/main/java/javaserver/ResponseBuilder.java

```diff
@@ -0,0 +1,66 @@
+package javaserver;
+
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class ResponseBuilder {
+	private String[] OK_ROOT_REQUESTS = {"GET"};
+	private String[] OK_FORM_REQUESTS = {"GET", "POST", "PUT"};
+	private String[] OK_METHOD_OPTIONS = {"GET", "HEAD", "POST", "OPTIONS", "PUT"};
```

Comment:
Did you name these variables in all caps because they are meant to be constants? If so, you'll need to add `static` and `final` keywords here.